sync3/handler/connstate.go

```diff
@@ -434,6 +440,35 @@ func (s *ConnState) buildRooms(ctx context.Context, builtSubs []BuiltSubscriptio
 	return result
 }
 
+func (s *ConnState) lazyLoadTypingMembers(ctx context.Context, response *sync3.Response) {
+	for roomID, typingEvent := range response.Extensions.Typing.Rooms {
+		if !s.lazyCache.IsLazyLoading(roomID) {
+			continue
+		}
+		room, ok := response.Rooms[roomID]
+		if !ok {
+			room = sync3.Room{}
+		}
```

Comment:
Doesn't `room` get set to a zero value anyway if `!ok`?  (Not that I mind us being explicit...)

Reply:
Yes